src/app/components/apparatus-entry/apparatus-entry-detail/apparatus-entry-detail.component.ts

```diff
@@ -0,0 +1,57 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ApparatusEntry, GenericElement, Reading } from '../../../models/evt-models';
+import { register } from '../../../services/component-register.service';
+import { EVTModelService } from '../../../services/evt-model.service';
+@Component({
+  selector: 'evt-apparatus-entry-detail',
+  templateUrl: './apparatus-entry-detail.component.html',
+  styleUrls: ['./apparatus-entry-detail.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+@register(ApparatusEntryDetailComponent)
+export class ApparatusEntryDetailComponent {
```

Comment:
this seems to contain duplicated code from the app/rdg parsers. If so, try to refactor this.